doc/user/events/widget.rst

```diff
@@ -293,6 +293,11 @@ with that information::
     </pretix-widget>
 
 This works for the pretix Button as well, if you also specify a product.
+
+As data-attributes are reactive, you can change them with JavaScript as well. Please note, that once the user
```

Comment:
Maybe make it sound less "obviously" to not suggest other attributes are reactive as well.
```suggestion
Data-attributes are reactive, therefore you can change them with JavaScript as well. Please note, that once the user
```

Reply:
Also, should we add a note that the mutation needs to be done on the div.pretix-widget-wrapper since the pretix-widget element goes away (at least in firefox)?

Reply:
Yes, a note would be great, thanks!